src/app/(light)/game/[eventName]/page.tsx

```diff
@@ -10,6 +10,7 @@ import Game from '@/components/Game/GameScreen'
 import GameNavBar from '@/components/NavBar/GameNavBar'
 import { getEventByEventName } from '@/lib/api/eventApi'
 import { Event } from '@/lib/types/types'
+import useSWR from 'swr'
 
 export default function GamePage(): JSX.Element {
     const [startTimer] = useState<number>(Date.now())
```

Comment:
bruker du fortsatt Date.now()? Kan ikke den slettes om du henter fra databasen? Eller er det for å regne på hvor lang tid spilleren bruker?